### DISCUSSION TITLE
Live Reloading doesn't work

### DISCUSSION THREAD

**Discussion by lups2000** [mitmproxy/mitmproxy#6664] [Q&A] [answered]
Hi there, I followed all the instructions in the readme to install mitmproxy and to run mitmweb, but i noticed that the live reload doesn't work, both on safari and chrome. I have to always clear the cache and then reload manually. I saw you recommend this "http://livereload.com/extensions/" (doesn't exist anymore?); meanwhile i installed another plugin for chrome but still doesn't work. Just to be sure, i ran on two terminals "npm run start" and "mitmweb", correct? Do you have any idea? Thank you! 🙏

**Comment by mhils** [comment, marked as answer]
Here's what works for me:

1. `npm run start` and `mitmweb` in two separate terminals. When editing script files, I get the following in the first console:

    ```
    [22:28:02] Starting 'scripts_dev'...
    [22:28:03] C:\Users\user\git\mitmproxy\mitmproxy\tools\web\static\app.js.map reloaded.
    [22:28:03] C:\Users\user\git\mitmproxy\mitmproxy\tools\web\static\app.js reloaded.
    [22:28:03] Finished 'scripts_dev' after 165 ms
    ```
2. I have [this](https://chromewebstore.google.com/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei) extension installed. Clicking on it enables it (black dot).
3. In the Chrome Dev Tools, I have "Disable cache (while DevTools open)" enabled.

Editing files then leads to immediate livereload for me.

**Reply by lups2000** [reply]
weird, I have the same exact settings but it doesn't work. By the way, I found a solution and by opening the development bar and disabling the cache , works! Thank you Maximilian for your help 👍
